src/system/LinkExternal/index.ts

```diff
@@ -0,0 +1,4 @@
+/**
```

Comment:
We are trying to avoid index files. Can you import LinkExternal directly from the LinkExternal.tsx in src/system/index.js?

Reply:
Yes, definitely!

Reply:
Made the change here: https://github.com/Automattic/vip-design-system/pull/390/commits/423eea1d22a4179a39e9be382b7bfefb5e5d62fb